upgrade/1.4.2/README.md

```diff
@@ -174,6 +174,43 @@ This step will install the `smart-mon` rpm on storage nodes, and reconfigure the
    /usr/share/doc/csm/scripts/operations/ceph/enable-smart-mon-storage-nodes.sh
    ```
 
+## Configure NCN nodes without restart
+
+This step will create an imperative CFS session that can be used to configure booted NCN nodes with updated sysctl values.
+
+1. (`ncn-m001#`) Create a new CFS Configuration Entry for the release
+
+   ```bash
+   COMMIT=`kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' 2>/dev/null | yq r -j - 2>/dev/null | jq --arg version "$CSM_RELEASE_VERSION" '. [$version].configuration.commit' | tr -d '"'`
```

Comment:
nit: I think `jq -r` might do what you're accomplishing with that final `tr -d`

Reply:
Jinx, you also owe me a coke @haasken-hpe .